types/proto/narwhal.proto

```diff
@@ -184,6 +184,8 @@ service WorkerToWorker {
 service WorkerToPrimary {
     // Sends a message
     rpc SendMessage(BincodeEncodedPayload) returns (Empty) {}
+
+    rpc WorkerInfo(Empty) returns (BincodeEncodedPayload) {}
```

Comment:
I was thinking the request would include the worker ID they are looking for and that worker address would be returned instead of the entire map but this works for now.

Reply:
Actually I take that back even on a failed request to a specific worker it would be pretty cheap to just request all the worker info and update the entire map in cache.

Reply:
Yeah for now it seems simplest to just request everything, its a small amount of data for now.